CONTRIBUTING.md

```diff
@@ -1,113 +1,29 @@
-# Contributing #
+Contributing
+============
 
-Information on contributing to this repo is in the [Contributing Guide](https://github.com/aspnet/Home/blob/dev/CONTRIBUTING.md) in the Home repo.
+The document covers the basic process for contributing to the [ASP.NET Core documentation site](https://docs.asp.net). See the [ASP.NET Core contributing guide](https://github.com/aspnet/Home/blob/dev/CONTRIBUTING.md) for information on contributing to ASP.NET Core source.
 
-The documentation is built using [Sphinx](http://sphinx-doc.org) and [reStructuredText](http://sphinx-doc.org/rest.html), and then hosted by [ReadTheDocs](http://aspnet.readthedocs.org).
+## Contributing process
 
-## Video: Getting Started ##
-[Watch a video](http://ardalis.com/contributing-to-asp-net-5-documentation) showing how to get started building the documentation locally.
+Small changes to one *.MD* file can be made in the browser by tapping the **Edit on GitHub** link in the upper right corner of the browser window. For new articles and changes involving more than one file:
```

Comment:
md should be lower case

In Docs the button just has "Edit", not "on GitHub", and in narrow windows it's in a collapsed-by-default "options" bar